src/shipit_code_coverage/tests/test_github.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import responses
+
+from shipit_code_coverage.github import GitHubUtils
+
+
+@responses.activate
+def test_get_commit(MERCURIAL_COMMIT, mock_get_commit):
```

Comment:
It would be useful to also add a test where the response is not 200

Reply:
This isn't trivial to test as the nested `get_commit` will be retried for some time as long as the request failed. Should I mock `wait_until`/`retry`?

Reply:
Right, we should mock them otherwise the test will take too long. Let's do it in a separate PR though, so we can merge these tests in the meantime!